CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v53.2.0...main)
 
+* General
+  * Experimental: Add configuration to add a `glean_timestamp` extra to all events ([#2513](https://github.com/mozilla/glean/issues/2513))
```

Comment:
Experimental, as in you are planning to test this with Nimbus? If so, that's nifty!

Reply:
Yup, that's the plan! (but also experimental as in: don't use it, we give no guarantees)